src/runtime_little_endian_heap.js

```diff
@@ -0,0 +1,74 @@
+/*  Stores value into the heap enforcing LE byte order. Uses DataView methods
```

Comment:
Can we avoid adding new `runtime_x.js` files, but instead add new `library_x.js` files? The existing `runtime_x.js` files are there for historical reasons, and we'd like to get new code towards a better structured and linked `library_x.js` files instead.

This way we will not first emit code in the build and then depend on a JS optimize to remove it when it sees no uses, but instead we will link the code in if it is actually used.

Reply:
Changed to library_little_endian_heap.js